docs/_data/nakadi-event-bus-api.yaml

```
@@ -574,13 +582,20 @@ paths:
           $ref: '#/definitions/EventTypeSchema'
         required: true
       responses:
+        '200':
+          description: |
+            Returns existing event type schema
```

```suggestion
            Existing event type schema, with the version.
```
Should probably also refer to a definition?